### PR TITLE
Notify users to read OpenTabletDriver's FAQ when tablet not detected

### PR DESCRIPTION
**Pull request by frenzibyte**
As multiple reports were submitted stating their tablets not getting detected by the game (e.g. https://github.com/ppy/osu/discussions/13652, https://github.com/ppy/osu/discussions/13807#discussioncomment-1164954), I've added a note under the "tablet not detected" message to read the OpenTabletDriver's FAQ of the running platform to troubleshoot the issue further, in hopes people read it rather than opening issues here.

CleanShot 2021-08-13 at 09 16 11

The note wording is open for suggestions, I'm not 100% about it but figured a PR would be the best place for this.

### CODE REVIEW

File: osu.Game/Overlays/Settings/Sections/Input/TabletSettings.cs [hunk 1]
```diff
@@ -73,12 +77,44 @@ private void load()
...
+                    Spacing = new Vector2(5f),
+                    Children = new Drawable[]
+                    {
+                        new OsuSpriteText
+                        {
+                            Anchor = Anchor.TopCentre,
+                            Origin = Anchor.TopCentre,
+                            Text = TabletSettingsStrings.NoTabletDetected,
+                        },
+                        new LinkFlowContainer
```
Comment (peppy) [inline osu.Game/Overlays/Settings/Sections/Input/TabletSettings.cs:95]: May want to make a class for this (to consolidate the visual behaviour with `SettingsItem.WarningText`).

File: osu.Game/Overlays/Settings/Sections/Input/TabletSettings.cs [hunk 2]
```diff
@@ -73,12 +77,44 @@ private void load()
...
+                            {
+                                t.NewLine();
+                                t.AddText("If your tablet is not getting detected properly,", s => s.Colour = colours.Yellow);
+                                t.NewParagraph();
+                                t.AddText("read ", s => s.Colour = colours.Yellow);
+                                t.AddLink("the driver's FAQ", LinkAction.External, RuntimeInfo.OS == RuntimeInfo.Platform.Windows
+                                    ? @"https://github.com/OpenTabletDriver/OpenTabletDriver/wiki/Windows-FAQ"
+                                    : @"https://github.com/OpenTabletDriver/OpenTabletDriver/wiki/Linux-FAQ");
+
+                                t.AddText(" to troubleshoot the problem further.", s => s.Colour = colours.Yellow);
```
Comment (peppy) [inline osu.Game/Overlays/Settings/Sections/Input/TabletSettings.cs:114]:
```suggestion
                                t.AddText("If your tablet is not detected, please read ", s => s.Colour = colours.Yellow);
                                t.AddLink("this FAQ", LinkAction.External, RuntimeInfo.OS == RuntimeInfo.Platform.Windows
                                    ? @"https://github.com/OpenTabletDriver/OpenTabletDriver/wiki/Windows-FAQ"
                                    : @"https://github.com/OpenTabletDriver/OpenTabletDriver/wiki/Linux-FAQ");
                                t.AddText(" for troubleshooting steps.", s => s.Colour = colours.Yellow);
```

It's not really a driver (misleading, I know), so we shouldn't be calling it that.

**Review by peppy — DISMISSED** [review]

File: osu.Game/Overlays/Settings/Sections/Input/TabletSettings.cs (hunk 1 above)
Comment (frenzibyte) [reply osu.Game/Overlays/Settings/Sections/Input/TabletSettings.cs:95]: `WarningText` is currently only a string, that's why I backed out from refactoring it to pass a `TextFlowContainer` just yet.

Comment (peppy) [reply osu.Game/Overlays/Settings/Sections/Input/TabletSettings.cs:95]: I meant the backing drawable that's displaying that string. Just making it a custom `LinkFlowContainer` with the correct styling applied. I think that's feasible?

Comment (frenzibyte) [reply osu.Game/Overlays/Settings/Sections/Input/TabletSettings.cs:95]: Aha, yeah should be feasible.

**Comment by bdach** [comment]
Was the link supposed to turn green?

2021-08-14-190442_492x90_scrot

I think you should be looking to set `defaultCreationParameters` on `SettingsNoticeText` rather than tint it all yellow.

**Comment by frenzibyte** [comment]
Right... *sigh*

**Comment by frenzibyte** [comment]
Had to make `OsuColour` a constructor parameter to be able to set it to `defaultCreationParameters`, doesn't seem bad compared to having the style in different places as originally.

**Comment by bdach** [comment]
It's a little unfortunate... You could also instantiate the `OsuColour` locally but that doesn't seem to great either. I don't really have any other bright ideas.

I think it's tolerable, but I'll wait for another approve to make sure.

**Review by bdach — APPROVED** [review]

**Review by smoogipoo — APPROVED** [review]
Lgtm

**Pull request merged**